forum/qa-plugin/tips-widget/tips-widget-widget.php

```diff
@@ -0,0 +1,48 @@
+<?php
+
+class tips_widget_widget
+{
+	public $random;
+	private $tips_list;
+	
+	function __construct()
+	{
+		if (!qa_opt('tips-enable'))
+			return;
+		
+		$this->tips_list = explode('!NEW!', qa_opt('tips-widget-content'));
+		$prev_random = isset($_COOKIE['prev_random']) ? $_COOKIE['prev_random'] : -1;
+		
+		do
+		{
+			$this->random = rand(0, count($this->tips_list) - 1);
+			
+		} while ($this->random == $prev_random);
+	}
+	
+	function allow_template($template)
+	{
+		if (!qa_opt('tips-enable'))
```

Comment:
brak {}

Reply:
poza tym mozna to w jednej linijce zmieścić bez dwóch returnów.

Reply:
Jak wyżej.